### ISSUE TITLE
[IPTV-Merge] name of channel is truncated or cutoff after merge

### ISSUE THREAD

**Issue by stuckinthe** [matthuisman/slyguy.addons#482]
On some of my TV Channel names, they get truncated or cutoff. Below is an example. I put the three lines in a file and after the merge it left me with the channel name of Texas

#EXTINF:-1 tvg-id="abc.kbmt.us" tvg-name="abc.kbmt.us" tvg-type="live" group-title="US Local" tvg-logo="https://media.host.live/abc.kbmt.us.png", Texas

```
#EXTM3U
#EXTINF:-1 tvg-id="abc.kbmt.us" tvg-name="abc.kbmt.us" tvg-type="live" group-title="US Local" tvg-logo="https://media.host.live/abc.kbmt.us.png",ABC (KBMT) Beaumont, Texas
http://localhost/abc.kbmt.us.m3u8
```

**Comment by matthuisman** [comment]
is it a valid .m3u8 if you have a comma in the name?

**Comment by matthuisman** [comment]
https://github.com/globocom/m3u8/blob/a4d6f2249a322b14230194c26d83114b4fc36f20/m3u8/parser.py#L255 splits on first comma, 

Ill try implement this fix tonight

**Comment by matthuisman** [comment]
fixed now with [v0.24.2](https://github.com/matthuisman/slyguy.addons/commit/d76f2ff17366aca1a97e406b260a03346c2c975e)
